movie-app-code-review-main/src/styles/style.css

```
@@ -0,0 +1,55 @@
+@import url(normalize.css);
+@import url('https://fonts.googleapis.com/css2?family=Inter:wght@100..900&display=swap');
+
+* {
+  box-sizing: border-box;
+}
+
+:root {
+  font-family: Inter, system-ui, Avenir, Helvetica, Arial, sans-serif;
+  line-height: 1.5;
+  font-weight: 400;
+
+  color-scheme: light dark;
+  color: rgba(255, 255, 255, 0.87);
+  background-color: #242424;
+
+  font-synthesis: none;
+  text-rendering: optimizeLegibility;
+  -webkit-font-smoothing: antialiased;
+  -moz-osx-font-smoothing: grayscale;
+  --black: black;
+  --black-light: rgba(0, 0, 0, 0.4);
+  --dark-grey: #2c2c2c;
+  --dark-yellow: #717148;
+  --grey: grey;
+  --white: #f0f0f0;
+  --perfect-white: #fff;
+  --blue: #2196f3;
+  --red: red;
+  --brown: brown;
+  --green-valid: aquamarine;
+  --green: #85cdca;
+  --light-purple: hsl(242, 61%, 76%);
+  --purple: hsl(242, 61%, 71%);
+  --light-green: #65c45d;
+  --orange: orange;
```

Use a consistent way to set colors.
for example:
```suggestion
  --black: #000000;
  --black-light: rgba(0, 0, 0, 0.4);
  --dark-grey: #2c2c2c;
  --dark-yellow: #717148;
  --grey: #808080;
  --white: #f0f0f0;
  --perfect-white: #ffffff;
  --blue: #2196f3;
  --red: #ff0000;
  --brown: #a52a2a;
  --green-valid: #7fffd4;
  --green: #85cdca;
  --light-purple: hsl(242, 61%, 76%);
  --purple: hsl(242, 61%, 71%);
  --light-green: #65c45d;
  --orange: #ffa500;
```